README.md

```diff
@@ -97,11 +97,99 @@ This will setup the build directories and compile. The compiled server will be i
+assets may cause issues due to incompatibility. The client is available to download
+[here](https://mega.nz/#F!8AkG3TzJ?UQc3zZKb).
+
+Run the installer and install the game to a desired location. While this project uses the assets
+from that client it does not use the binaries. For full compatibility you will need to download
+the latest client binaries from the project's [Releases Page](https://github.com/dev-osrose/osIROSE-new/releases)
+
+After downloading the client files, copy them to the `ROSE_112_112NA` installation directory (default:
+`C:\Program Files (x86)\TriggerSoft\Rose Online`) and overwrite the existing files.
+
+ Connecting to the server requires using a special command from the command line, like so:
+
+`trose.exe @TRIGGER_SOFT@ _server 127.0.0.1` (replace the IP if not running locally
+
+For convenience, this command can be saved to a file with the extension `.cmd` in the same folder
+as `trose.exe`. Then, the game can be launched by double clicking this file.
+
+Database Setup
+---------------
+The server requires a [MySQL](https://mysql.com/) (or [MariaDB](https://mariadb.com/)) database server.
+
+To setup the database, first create a new database (e.g. `osirose`). You will then need to execute the
+`Database/osirose.sql` script to generate the database schema and the `Database/item_db.sql` to
+populate the item database. This can be done using a database tool or from the command line like so: 
+```
+mysql -u<user> -p<password> <database_name> < Database/osirose.sql
+mysql -u<user> -p<password> <database_name> < Database/item_db.sql
+```
+
+**NOTE**: The item table was generated using client v112_112na. If you are using a different client,
+please generate your own table to match the information you need.
+
+After setting up the database, two special users need to be created for the login and character
+servers. They can be created by executing the following SQL commands:
+
+```
+call create_account("login", "098f6bcd4621d373cade4e832627b4f6");
+call create_account("char", "098f6bcd4621d373cade4e832627b4f6");
+update accounts set account_type="system" where username="login";
+update accounts set account_type="system" where username="char";
+```
+
+The value `098f6bcd4621d373cade4e832627b4f6` is an MD5 hash of the default server passwords created
+in the default `server.json` file. It should match the following sections:
+
+```
+    ...
+    "charServer":  {
+        ...
+        "loginUser":         "login",
+        "loginPassword":     "098f6bcd4621d373cade4e832627b4f6",
+        ...
+    },
+    "mapServer":   {
+        ...
+        "charUser":     "char",
+        "charPassword": "098f6bcd4621d373cade4e832627b4f6",
+        ...
+    },
+    ...
+```
+
+To use your own password, generate an [MD5 hash](https://www.md5hashgenerator.com) of your password 
+and update the accounts database and the `server.json` file.
+
+Finally, we need to create an account to join the game with. Run the `create_command` procedure
```

Comment:
you mean `create_account` instead of `create_command`

Reply:
Updated and rebased